src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/data/VariantType.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr.scalable.data;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+
+/**
+ * Logic for determining variant types was retained from VQSR.
+ */
+public enum VariantType {
+    SNP,
+    INDEL;
+
+    public static boolean checkVariantType(final VariantContext vc,
+                                           final VariantContext resourceVC) {
+        switch (resourceVC.getType()) {
+            case SNP:
+            case MNP:
+                return getVariantType(vc) == SNP;
+            case INDEL:
+            case MIXED:
+            case SYMBOLIC:
+                return getVariantType(vc) == INDEL;
+            default:
+                return false;
+        }
+    }
+
+    public static VariantType getVariantType(final VariantContext vc) {
+        if (vc.isSNP() || vc.isMNP()) {
```

Comment:
Yeah, seems like we could use a similar switch statement. Also not sure why e.g. structural indels are treated differently. But I'll just mention this in the issue and leave as is for now, just in case there is any weird edge-case behavior that we'd change if we changed this code.